janusgraph-test/src/main/java/org/janusgraph/diskstorage/indexing/IndexProviderTest.java

```diff
@@ -84,35 +86,54 @@ public KeyInformation get(String key) {
         };
     }
 
-    public static final Map<String,KeyInformation> getMapping(final IndexFeatures indexFeatures) {
+    public static final Map<String,KeyInformation> getMapping(final IndexFeatures indexFeatures, final String englishAnalyzerName, final String keywordAnalyzerName) {
         Preconditions.checkArgument(indexFeatures.supportsStringMapping(Mapping.TEXTSTRING) ||
                 (indexFeatures.supportsStringMapping(Mapping.TEXT) && indexFeatures.supportsStringMapping(Mapping.STRING)),
                 "Index must support string and text mapping");
         return new HashMap<String,KeyInformation>() {{
-            put(TEXT,new StandardKeyInformation(String.class, Cardinality.SINGLE, new Parameter("mapping",
-                    indexFeatures.supportsStringMapping(Mapping.TEXT)?Mapping.TEXT:Mapping.TEXTSTRING)));
+            put(TEXT,new StandardKeyInformation(String.class, Cardinality.SINGLE,
+                    indexFeatures.supportsStringMapping(Mapping.TEXT)?Mapping.TEXT.asParameter():Mapping.TEXTSTRING.asParameter()));
```

Comment:
I like how you cleaned these statements up. Going a little further how about `(indexFeatures.supportsStringMapping(Mapping.TEXT)?Mapping.TEXT:Mapping.TEXTSTRING).asParameter())`?